leaf_size: 0.0025
model_folder: ""
number_of_average_clouds: 1
number_of_median_clouds: 1
z_threshold: 0.01
planarSegmentationTolerance: 0.02
min_number_of_inliers: 15000
xmin: -0.2
xmax: 0.2
ymin: -0.1
ymax: 0.15
zmin: 0
zmax: 1.0

bin_size: 0.015
inlier_dist_segmentation: 0.01
segmentation_inlier_ratio: 0.2
max_number_of_instances: 6
max_fitness_score: 0.0003
inlier_dist_icp: 0.003
icp_inlier_threshold: 0.7
min_inlier_ratio_validation: 12
inlier_dist_validation: 0.0025

full_display: true
save_clouds: true
use_all_points: true
use_hough: false

package_path: branch_surface
point_cloud_topic: /camera/depth_registered/points